experimental/docker-stacks.md

```diff
@@ -0,0 +1,98 @@
+# Docker Stacks
+
+## Overview
+
+Docker Stacks are an experimental feature introduced in Docker 1.12, alongside
+the new concepts of Swarms and Services inside the Engine.
```

Comment:
But what are they? This just talks about using them but not about what they are, what use cases they're trying to solve.  Do we have any docs on services too?